pycloudlib/lxd/cloud.py

```diff
@@ -567,45 +572,68 @@ def _image_info(self, image_id, image_hash_key=None):
             image_id, image_hash_key=self.DISK1_HASH_KEY
         )
 
+    def _lxc_image_info(self, image_id: str) -> dict:
+        """Return a dict of the output of ``lxc image info <image_id>``.
+
+        Args:
+            image_id: string, [<remote>:]<image identifier>, the image to
+                      return the image info dict for
+
+        Returns:
+            A dict produced by loading the YAML emitted by ``lxc image info
+            <image_id>``, or the empty dict if either the command or YAML load
+            fails.
+        """
+        raw_image_info = subp(["lxc", "image", "info", image_id], rcs=())
+        if raw_image_info.ok:
+            try:
+                return yaml.safe_load(raw_image_info)
+            except yaml.error.YAMLError:
```

Comment:
Is this right? My editor says this is an error and I think it's just `yaml.YAMLError` rather than `yaml.error.YAMLError`